RELEASE_GUIDE.md

```diff
@@ -0,0 +1,115 @@
+# SOP: OpenSearch-Benchmark Release Guide
+
+- [Overview](#overview)
+- [Branches](#branches)
+    - [Releasing Branches](#release-branches)
+    - [Feature Branches](#feature-branches)
+- [Release Labels](#release-labels)
+- [Prerequisites](#prerequisites)
+- [Release new version of OpenSearch Benchmark to PyPi, Docker, and ECR](#release-new-version-of-opensearch-benchmark-to-pypi-docker-and-ecr)
+- [Error Handling](#error-handling)
+
+## Overview
+This document explains the release strategy for artifacts in this organization.
+
+## Branches
+
+### Release Branches
+
+Given the current major release of 1.0, projects in this organization maintain the following active branches.
+
+* **main**: The next `1.x` release. This is the branch where all merges take place and code moves fast.
+* **1.0**: The _current_ release. In between minor releases, only hotfixes (e.g. security) are backported to `1.0`.
+
+Label PRs with the next major version label (e.g. `2.0.0`) and merge changes into `main`. Label PRs that you believe need to be backported as `1.x` and `1.0`. Backport PRs by checking out the versioned branch, cherry-pick changes and open a PR against each target backport branch.
+
+### Feature Branches
+
+Do not creating branches in the upstream repo, use your fork, for the exception of long lasting feature branches that require active collaboration from multiple developers. Name feature branches `feature/<thing>`. Once the work is merged to `main`, please make sure to delete the feature branch.
```

Comment:
with the exception 
Name feature branches `feature/<FEATURE>`.

Once the feature branch has been merged into `main`, make sure to delete it.